docs/agents/sematext-agent/agent-troubleshooting.md

```diff
@@ -0,0 +1,19 @@
+title: Agent Troubleshooting
+
+If you are experiencing issues with the Sematext Agent, there are some things you can try.
+
+1. Try restarting the agent. Specific instructions on how to do so depending on your environment can be found [here](https://sematext.com/docs/agents/sematext-agent/starting-stopping).
```

Comment:
Before I restart, are there agent log files somewhere where I could look?  Let's provide that info.  Are there any specific things to look for other then things like 'error'?

Reply:
Yago and I have talked about this and we can't really think of anything specific to look for other than verifying Vector and Telegraf are running. 
I'll add to check for errors in the log in case there's an error with permissions or configuration files